client/ReviewList.jsx

```diff
@@ -1,22 +1,57 @@
 import React from 'react';
 import ReviewListItem from './ReviewListItem.jsx';
 import styles from './styles/reviewListStyle.css'
+import { faChevronRight, faChevronLeft } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 
 class ReviewList extends React.Component {
   constructor(props) {
     super(props);
   }
   render() {
     const reviews = this.props.reviews;
-    return(
-      <ul className={styles.unorderedList}>
-        {reviews.map((review) =>
-          <ReviewListItem
-          key={review.guestName}
-          value={review}
-          />
-        )}
-      </ul>
+    return (
+      <div>
+        <ul className={styles.unorderedList}>
+          {reviews.map((review) =>
+            <ReviewListItem
+              key={review.guestName}
+              value={review}
+            />
+          )}
+        </ul>
+        <div className={styles.buttonContainer}>
+          <div className={styles.chevronHolder}>
+            <button onClick={this.props.clickPrevious}>
+              <div>
+                <FontAwesomeIcon icon={faChevronLeft} size="lg" className={styles.leftChevron} />
+              </div>
+            </button>
+          </div>
+          {/* <div className={styles.numberButton}>
```

Comment:
this is commented out, but if you wanted to render all the numbers you could consider making an array of page numbers and mapping over the numbers you would like to display (similar to what you did above for reviews)

Reply:
That's a great idea. I think along with refactoring my codebase to be more pleasant this is also a goal if I have time (pagination)